projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.pipes.ts

```diff
@@ -58,7 +63,10 @@ export class IgxPivotRowExpansionPipe implements PipeTransform {
         _pipeTrigger?: number,
         __?,
     ): any[] {
-        const pivotKeys = config.pivotKeys || { aggregations: 'aggregations', records: 'records', children: 'children', level: 'level' };
+        const pivotKeys = config.pivotKeys || {
```

Comment:
I think it would be better if we have the default values stored somewhere as a global variable, since I saw the same object in multiple places and in different files.

Reply:
I think after merge conflicts are resolved, these object will occur only in one place - in its getter. There were additional PR targeting this issue

Reply:
I don't see this being addressed in the `pivot-grid-master` branch unless there is another PR